trollmoves/movers.py

```diff
@@ -329,6 +329,12 @@ def copy(self):
             ssh_connection.close()
             raise
 
+        try:
+            ssh_connection.exec_command("mkdir -pv {}".format(self.destination.path))
```

Comment:
This is a nice addition! Should there be a explicit config option which, if set to `True` (or equivalent) enables the feature? In some cases this as a default behaviour can hide some quite significant errors :-)

Reply:
Yes, I did think about something like that, but forgot about it.

So default should be: Do not create remote dir?
This to make it clear to the user if he needs this feature?

Reply:
Yes, I think remote dir creation should be turned off by default. Easier to catch typos and such if the feature isn't needed.

Reply:
Hm, looking closely on this, its actually not needed. because there is a `create_local_dir` in the client.py called in `request_push`